.github/workflows/windows-periodic.yml

```diff
@@ -6,6 +6,9 @@ on:
   schedule:
   - cron: "0 1 * * *"
   workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/windows-periodic.yml'
```

Comment:
I'm a bit lost here.

> there is no way to run this via pull_request trigger as the secrets are not visible for these actions. It's basically a security risk if they would, as one could easily write a PR that can display all the secrets in this repo. The only way this would work is if the PR is issued by someone who already has write access to the repo.

If so, can we keep this section?

Reply:
We can remove it, I'll update the PR. I thought I did, but apparently didn't actually push changes to my branch :(